src/xrCore/xrDelegate/xrDelegateBinder.h

```diff
@@ -154,5 +154,5 @@ static xrDelegateArgumentsTypes<Args...> BindDelegateArgs(Args...args)
 template<typename ... Args>
 static xrDelegateArgumentsTypes<Args...>* BindDelegateArgsPtr(Args&&...args)
 {
-    return new xrDelegateArgumentsTypes<Args...>(std::forward<Args>(args)...);
+    return xr_new<xrDelegateArgumentsTypes<Args...>(std::forward<Args>(args)...)>();
```

Comment:
It's wrong. The right replacement is:
```suggestion
    return xr_new<xrDelegateArgumentsTypes<Args...>>(std::forward<Args>(args)...);
```